airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -115,10 +141,154 @@ public TemporalClient(final WorkflowClient client, final Path workspaceRoot) {
             connectionId));
   }
 
+  public void submitConnectionUpdaterAsync(final UUID connectionId) {
+    log.info("Starting the scheduler temporal wf");
+    final ConnectionUpdaterWorkflow connectionUpdaterWorkflow = getWorkflowOptionsWithWorkflowId(ConnectionUpdaterWorkflow.class,
+        TemporalJobType.CONNECTION_UPDATER, "connection_updater_" + connectionId);
```

Comment:
nit: the magic string here for constructing the name should probably be a function

Reply:
Done